test/helpers/convertDataToBytes.js

```diff
@@ -0,0 +1,12 @@
+const convertDataToBytes = (data) => {
+  let concatenatedString = ''
+
+  Object.keys(data).forEach((key) => {
```

Comment:
FYI if you're only using the value, you can use [`Object.values()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values).

(Not saying you should to change this, just letting you know in case you were unaware.)